Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN python3 -m pip install /workspace/vllm-ascend/ --extra-index https://downloa
 # Install torch-npu
 RUN bash /workspace/vllm-ascend/pta_install.sh
 
-# Install modelscope
-RUN python3 -m pip install modelscope ray
+# Install modelscope (for fast download) and ray/net-tools (for multinode)
+RUN python3 -m pip install modelscope ray net-tools
```

Comment:
I assuem the net-tools should be install by `apt`? @MengqingCao

Reply:
oh my bad